lib_comfyui/external_code.py

```diff
@@ -91,12 +91,12 @@ def get_workflow_display_names(tabs: Tabs = ALL_TABS) -> List[str]:
     return [workflow.display_name for workflow in get_workflows(tabs)]
 
 
-def get_default_workflow_json(iframe_id: str) -> dict:
+def get_default_workflow_json(workflow_id: str) -> dict:
```

Comment:
in my view, again, this should stay named as `iframe_id` (or renamed as `workflow_type_id`)